com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolumeProfile.cs

```diff
@@ -50,9 +48,10 @@ public enum CellSize
         [SerializeField]
         Version version = CoreUtils.GetLastEnumValue<Version>();
 
-        public int maxSubdivision => Mathf.CeilToInt(Mathf.Log((float)cellSize / brickSize, 3));
+        // We don't count the first subdivision because it have the same size as the cell so we add one.
```

Comment:
Is this already valid before your subdiv changes?

Reply:
With this PR yes, adding one subdiv level will let the structure subdivide up to the cell level, and because the cell size is compatible with brick size (brickSize * 3^subdivLevel) now it's safe to do that (bricks won't be generated outside of the cell)